### PR TITLE
Allow to override all SecDefaultAction attributes

### PR DESCRIPTION
**Pull request by sathieu**

### CODE REVIEW

File: spec/classes/mod/security_spec.rb [hunk 1]
```diff
@@ -47,10 +47,13 @@
 
     describe 'with parameters' do
       let :params do
-        { :audit_log_parts => "ABCDZ"
+        {
+          :audit_log_parts  => "ABCDZ",
+	  :secdefaultaction => "deny,status:406,nolog,auditlog"
```
Comment (tphoney) [inline spec/classes/mod/security_spec.rb]: this needs aligned

File: spec/classes/mod/security_spec.rb [hunk 2]
```diff
@@ -100,10 +103,13 @@
 
     describe 'with parameters' do
       let :params do
-        { :audit_log_parts => "ACEZ"
+        {
+          :audit_log_parts  => "ABCDZ",
+	  :secdefaultaction => "deny,status:406,nolog,auditlog"
```
Comment (tphoney) [inline spec/classes/mod/security_spec.rb]: this needs aligned too

**Comment by tphoney** [comment]
@sathieu this code looks pretty good, thanks for updating the readme as well. There is some minor alignment that needs corrected.

**Comment by DavidS** [comment]
Closing to prefer rebased and unified #1494

**Pull request closed without merging**